<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>officPanel</class>
 <widget class="QMainWindow" name="officPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>officPanel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="showOrdersBtn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Показать все заказы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameFood">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Введите блюдо</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newOrderBtn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Подтвердить новый заказ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="orderPayed">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Оплаченный заказ (номер)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="orderPayedBtn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выбранный заказ оплачен</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
